app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+# Controller for Registrations under Devise
+class RegistrationsController < Devise::RegistrationsController
+  protected
+
+  def sign_up_params
+    params.require(:user).permit(:email, :password, :password_confirmation,
+                                 :first_name, :last_name)
```

Comment:
Yeah, I suppose we could put gender in, I left intent and role out b/c I wasn't sure what we wanted to ultimately do with the sign up